pytket/extensions/qiskit/backends/ibm.py

```diff
@@ -124,19 +125,48 @@ def _save_ibmq_auth(qiskit_config: Optional[QiskitConfig]) -> None:
     token = None
     if qiskit_config is not None:
         token = qiskit_config.ibmq_api_token
-    if not IBMQ.active_account():
-        if IBMQ.stored_account():
-            IBMQ.load_account()
+    if token is None and os.getenv("PYTKET_REMOTE_QISKIT_TOKEN") is not None:
```

Comment:
I'm assuming we are keeping this workaround with the remote token in the released version and we have not found a better way to avoid this yet?

Reply:
I hadn't seen this change, why was it needed? Seems like the wrong place to be fixing a problem with our CI.